### PR TITLE
Fixing Broken Link

### PR DESCRIPTION
**Pull request by TokyoScarab**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/2417

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by CarolynRowe** [comment]
@scanum:  Please use this link instead:  https://docs.microsoft.com/en-us/SkypeForBusiness/hybrid/msft-telephony-solutions

Then I will merge the PR.  Thanks!

File: Skype/SfbServer/plan-your-deployment/enterprise-voice-solution/enterprise-voice-solution.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ description: "Summary: Learn about your options for planning a unified voice and
   
 Skype for Business Server offers two on-premises Enterprise Voice options—Enterprise Voice and Call Via Work—for you to integrate the telephone system of your organization with your Skype for Business Server deployment, making for a true unified communications solution. Both options enable users to use their Skype for Business client to initiate voice calls with other users, both inside and outside of your organization.
   
-You can also take advantage of Microsoft Cloud PBX services, including PBX services and PSTN calling, by connecting your on-premises telephony infrastructure to services provided by Skype for Business Online. For more information, see [Plan your Phone System in Office 365 (Cloud PBX) solution](../../skype-for-business-hybrid-solutions/plan-your-phone-system-cloud-pbx-solution/plan-your-phone-system-cloud-pbx-solution.md).
+You can also take advantage of Microsoft Cloud PBX services, including PBX services and PSTN calling, by connecting your on-premises telephony infrastructure to services provided by Skype for Business Online. For more information, see [Cloud voice in Microsoft Teams](https://docs.microsoft.com/microsoftteams/cloud-voice-landing-page).
```
Comment (nenonix) [inline Skype/SfbServer/plan-your-deployment/enterprise-voice-solution/enterprise-voice-solution.md]:
```suggestion
You can also take advantage of Microsoft Cloud PBX services, including PBX services and PSTN calling, by connecting your on-premises telephony infrastructure to services provided by Skype for Business Online. For more information, see [Microsoft Telephony Solutions](https://docs.microsoft.com/SkypeForBusiness/hybrid/msft-telephony-solutions).
```

**Pull request merged**